### PR TITLE
refactor menu from admin dashboard

### PR DESCRIPTION
**Pull request by johanna-rodriguez**
## Description

Refactor Admin dashboard component to render the components of the menu keeping the state of itself. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes
https://www.loom.com/share/8be69ec889cb473c9cc3767ba8890e63

### CODE REVIEW

**Review by Broveku — APPROVED** [review]
code looks great, as does the routing handling the admin dashboard

**Review by VAIBHAVIBALAR — APPROVED** [review]
This is a great way to refactor components such that we don't re-render the components again
Overall superclean code.

**Review by rickmansfield — APPROVED** [review]
Looks great we need this sooner than later

**Comment by nhamilton1** [comment]
Awesome work, this is going to help immensely!

**Comment by ayv8er** [comment]
This was a valuable addition for the next team to work off of!

**Review by cyberkade — APPROVED** [review]
Clean routing, let's get this merged!

File: src/components/pages/Availability/Availability.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import RenderDashboardContainer from '../../utils/renderDashboardComponent';
 
 export function Availability({ LoadingComponent }) {
   return (
     <>
-      <RenderDashboardContainer content="xyz Availability" />
+      <h1>This is a test render from Availavility</h1>
```
Comment (paulstgermain) [inline src/components/pages/Availability/Availability.js:6]: Slight typo here.

**Pull request merged**